README.md

```diff
@@ -1,30 +1,183 @@
-book [Hello-World]
+<a name="readme-top"></a>
 
+<div align="center">
+  
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
 
-[your_project__name] is a set up project
+  <h3><b>Microverse README Template</b></h3>
```

Comment:
- [ ] [OPTIONAL]  Kindly, delete this part as well 😃. This for a good look and professional README 👍 you can replace it with your own logo if you have one.

![dele](https://user-images.githubusercontent.com/57408419/219443004-0e29f751-292b-4534-aa0b-e285a6e685a2.jpg)